MediaProcessor/src/DeepFilterCommandBuilder.cpp

```diff
@@ -0,0 +1,43 @@
+#include "DeepFilterCommandBuilder.h"
+#include <stdexcept>
+#include <sstream>
+
+namespace MediaProcessor {
+
+DeepFilterCommandBuilder& DeepFilterCommandBuilder::setInputAudio(const std::string& inputAudioPath) {
+    m_inputAudioPath = inputAudioPath;
+    addFlag("--input");
```

Comment:
Currently we are passing the input file path as the argument [here](https://github.com/omeryusufyagci/fast-music-remover/blob/e04313276c5149d66853c8864a546eeda68e4fa2/MediaProcessor/src/AudioProcessor.cpp#L153) without an `--input` flag, is this supported as well? 
If not we can simply pass  this path via the `addArgument()` method. 
Could you please check?

Reply:
Hey, thanks for pointing this out, it looks like it is not necessary so we can go ahead and just use the addArgument() method!